flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/source/reader/external/IncrementalSourceStreamFetcher.java

```diff
@@ -85,6 +87,7 @@ public void submitTask(FetchTask<SourceSplitBase> fetchTask) {
                     try {
                         streamFetchTask.execute(taskContext);
```

Comment:
Should we need to add `this.currentTaskRunning = false;` here ?

Reply:
I think this is to ensure the task running is set properly after exception 

e.g., MySQL connector has a similar logic here
https://github.com/ververica/flink-cdc-connectors/blob/master/flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/debezium/reader/BinlogSplitReader.java#L130